### DISCUSSION TITLE
How does Relays know about new PDSes?

### DISCUSSION THREAD

**Discussion by AnthonyAkentiev** [bluesky-social/atproto#2014] [Clarifications and "Why?"] [answered]
I have a question on "federation" of Bluesky.

How does Relays/AppViews know about new PDSes?
Does new PDS register in the relay somehow? Is it a permissionless call (anyone can do it) or does it require some "admin" authorisation? 

Thank you

**Comment by snarfed** [comment, marked as answer]
PDSes call [`com.atproto.sync.requestCrawl`](https://github.com/bluesky-social/atproto/blob/main/lexicons%2Fcom%2Fatproto%2Fsync%2FrequestCrawl.json) on a given relay to register with it. Right now that call is permissionless.

**Comment by AnthonyAkentiev** [comment]
@snarfed Thank you
